Directory.Build.props

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <Deterministic>true</Deterministic>
```

Comment:
This should only be set on the CI AFAIK

Reply:
I've not yet encountered a reason to not simply apply it everywhere, but my mindset _does_ lean towards deterministic systems in-general. In principal I like it here because it makes me more confident the build working on my local machine will produce a working result when running in a CI pipeline.